compiler/util/ir_util.py

```diff
@@ -252,7 +256,7 @@ def _find_path_in_enumeration(path, type_definition):
 
 
 def _find_path_in_parameters(path, type_definition):
-  if len(path) > 1:
+  if len(path) > 1 or not type_definition.HasField("runtime_parameter"):
```

Comment:
`if len(path) > 1 or not type_definition.runtime_parameter:`

?

Reply:
That will crash if `runtime_parameter` is not part of `type_definition`, `HasField` will not.